services/website-api/src/controller/user/UserController.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from '@tsoa/runtime';
 import { UserRoles } from '@donategifts/common';
 import { UserService } from '@donategifts/user';
+import { IAPIUser } from './types/IAPIUser';
```

Comment:
why APIUser?

Reply:
because tsoa can't inherit types which aren't marked as tsoaModel, i didn't mark the User because it holds a TypeObjectId which can't be resolved from tsoa
that's why we need a separate type for that
will fix itself once we move to typeorm and get rid of all the mongo stuff